packages/apps/human-app/frontend/src/pages/worker/jobs/components/available-jobs/mobile/available-jobs-network-filter-mobile.tsx

```diff
@@ -1,15 +1,17 @@
 /* eslint-disable camelcase --- ... */
-import { chains } from '@/smart-contracts/chains';
+import { useGetAllNetworks } from '@/hooks/use-get-all-networks';
 import { Filtering } from '@/components/ui/table/table-header-menu.tsx/filtering';
 import { useJobsFilterStore } from '@/hooks/use-jobs-filter-store';
 
-const allNetworks = chains.map(({ chainId, name }) => ({
-  option: chainId,
-  name,
-}));
+interface AvailableJobsNetworkFilterProps {
+  chainIdsEnabled: number[];
+}
 
-export function AvailableJobsNetworkFilterMobile() {
+export function AvailableJobsNetworkFilterMobile({
```

Comment:
This component is used in `DrawerMobile` and due to new props build fails.

Luckily, `DrawerMobile` is not used anywhere, so probably we just need to remove it